src/app/components/list-cities/list-cities.scss

```diff
@@ -0,0 +1,93 @@
+$salmon: rgb(255, 82, 82);
+
+main-page {
+    .container-fluid {
+        padding-top: 10%;
+        margin-left: 2%;
+        margin-right: 2%;
+    }
+
+    ul {
+        padding: 0;
+        margin: 0;
+        list-style: none;
+
+        display: -webkit-box;
+        display: -moz-box;
+        display: -ms-flexbox;
+        display: -webkit-flex;
+        display: flex;
```

Comment:
these styles for ul element are duplicated, so try to declare one time and reuse it
additionally. I think that assign five times display properties have no sense, I mean, does not work with just flex??